src/home_robot/home_robot/mapping/semantic/categorical_2d_semantic_map_module.py

```diff
@@ -309,7 +309,7 @@ def _update_local_map_and_pose(
 
         agent_view = torch.zeros(
             batch_size,
-            obs_channels,
+            5 + self.num_sem_categories,
```

Comment:
This was a bug. This happened to work because `obs_channels` was same as the number of channels in local map earlier.
`obs` has the egocentric observations, while `agent_view` here is supposed to contain the top-down local maps.